docs/references/_attachments/ic.did

```diff
@@ -53,6 +53,10 @@ type change_details = variant {
         canister_version : nat64;
         snapshot_id : snapshot_id;
         taken_at_timestamp : nat64;
+        source : variant {
```

Comment:
Changelog should be updated (shortly - to avoid conflicts) before merging this PR. The PR should be merged when the implementation of the feature is rolled out to all subnets.